pupil-spa/src/app/login-failure/login-failure.component.html

```diff
@@ -0,0 +1,45 @@
+<app-page-modifications>
```

Comment:
What is this tag? Why is it here?

Reply:
@jon-shipley this tag provides necessary styling for font size and colour contrasts, although it makes me think that at that stage they are not needed since the personalised choices are only fetched upon successful login.

Reply:
@jon-shipley removed along with the relevant AA classes.